modules/core/site.js

```diff
@@ -2399,3 +2410,74 @@ const observeMessageTextMutationAndHandleExternalResources = (inline) => {
         });
     }
 };
+
+function setupActionSchedule(callback) {
+    $('.nexter_date_picker').off('click').on('click', function (e) {
+        document.querySelector('.nexter_input_date').showPicker();
+    });
+
+    $('.nexter_date_helper').off('click').on('click', function (e) {
+        e.preventDefault();
+        $('.nexter_input').val($(this).attr('data-value')).trigger('change');
+    });
+
+    $('.nexter_input_date').off('input').on('input', function (e) {
+        let now = new Date();
+        now.setMinutes(now.getMinutes() + 1);
+        $(this).attr('min', now.toJSON().slice(0, 16));
+        if (new Date($(this).val()).getTime() <= now.getTime()) {
+            $('.nexter_date_picker').css('border', '1px solid red');
+        } else {
+            $('.nexter_date_picker').css({ 'border': 'unset', 'border-top': '1px solid #ddd' });
+        }
+    });
+
+    $('.nexter_input_date').off('change').on('change', function (e) {
+        if ($(this).val() && new Date().getTime() < new Date($(this).val()).getTime()) {
+            $('.nexter_input').val($(this).val()).trigger('change');
+        }
+    });
+
+    $('.nexter_input').off('change').on('change', callback);
```

Comment:
The `off → on` pattern is problematic. If you want to clean up event handlers, page handlers allow you to return a function that cleans up states or executes any necessary code when a page is unmounted. For example, see line 10 in [this file](https://github.com/cypht-org/cypht/blob/master/modules/imap/js_modules/route_handlers.js).

However, cleanup is not necessary here because each call to your handler (`setupActionSchedule`) will apply to a new and unique instance of the main element in the DOM.